extensions/control-plane/api/management-api/catalog-api/src/main/java/org/eclipse/edc/connector/api/datamanagement/catalog/CatalogApiExtension.java

```diff
@@ -15,7 +15,7 @@
 package org.eclipse.edc.connector.api.datamanagement.catalog;
```

Comment:
The package should be renamed as well. As far as I see it's `management` in the api configuration module, but none of the others.

Reply:
good catch!